scripts/docker/entrypoint_ci.sh

```diff
@@ -221,6 +221,12 @@ if [[ ${SKIP_ENVIRONMENT_INITIALIZATION=} != "true" ]]; then
         ssh-keyscan -H localhost >> ~/.ssh/known_hosts 2>/dev/null
     fi
 
+    sudo apt-get update && sudo apt-get install -y vsftpd
+    mkdir /ftphome
+    sudo chown airflow:airflow /ftphome
+    echo "write_enable=YES" >> /etc/vsftpd.conf
+    sudo service vsftpd restart
```

Comment:
Here I pull up the local FTP server for testing. I am not sure if this is the appropriate place to do this, but I did not find any other part of the codebase where this should be done.

Reply:
I’d prefer simply mocking `ftplib` for testing.

Reply:
I see, but I saw from the SFTPOperator tests that they also pull up an local ssh server and use it directly for testing as opposed to mocking the server (tests [here](https://github.com/apache/airflow/blob/main/tests/providers/sftp/operators/test_sftp.py)). I thought that, for consistency and accuracy purposes, it would make sense to do the same for the FTPOperator.

Reply:
I’d argue there’s a limit on being thorough and the SFTP tests are probably too much trouble to be worthwhile. They are already there so there’s less reason to rewrite them, but it’s better for new code to be tested in a more lightweight fashion.

Reply:
That make sense. I have replaced the tests with more lightweight unit tests (see [here](https://github.com/RachitSharma2001/airflow/blob/main/tests/providers/ftp/operators/test_ftp.py)). Let me know if these tests make sense, and if there are any other changes that should be made.